src/pf-dropdown/index.html

```diff
@@ -0,0 +1,115 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>pf-dropdown example</title>
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.13.0/css/patternfly.css">
+  <link rel="stylesheet" href="/dist/css/patternfly.css">
+  <link rel="stylesheet" href="/dist/css/patternfly-webcomponents.css">
+  <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+  <script src="/dist/js/patternfly.js"></script>
+</head>
+
+<body>
+  <div class="container">
+    <div class="page-header">
+      <h1>Dropdown</h1>
+    </div>
+
+    <!-- Dropdown Button -->
+    <pf-dropdown id="dropdown1">
+      <div class="dropdown">
```

Comment:
This div should not be needed, use pf-dropdown element for that class instead (and ensure it styles the same as div) ;)